capstone/src/main/java/com/google/sps/servlets/clubs/ClubServlet.java

```diff
@@ -124,4 +152,40 @@ private PreparedQuery retrieveClub(HttpServletRequest request, DatastoreService
     PreparedQuery prepared = datastore.prepare(query);
     return prepared;
   }
+
+  /** Return the Calendar ID after creating a calendar for the given club name. */
+  public String createCalendar(String clubName, Calendar service)
+      throws IOException, GeneralSecurityException {
+    com.google.api.services.calendar.model.Calendar calendar =
+        new com.google.api.services.calendar.model.Calendar()
+            .setSummary(clubName + " Calendar")
+            .setTimeZone("America/Los_Angeles");
+    String createdCalendarId = service.calendars().insert(calendar).execute().getId();
+
+    // TODO: set up permissions for club members (read only) and officers (read and write)
+    // Enable reader permission for user
+    // AclRule rule =
+    //     new AclRule().setScope(new
+    // Scope().setType(SCOPE_TYPE)).setRole(USER_CALENDAR_PERMISSIONS);
+    // service.acl().insert(createdCalendarId, rule).execute();
+    return createdCalendarId;
+  }
+
+  private Calendar getCalendarService() throws IOException, GeneralSecurityException {
+    String userId = UserServiceFactory.getUserService().getCurrentUser().getUserId();
+    Credential credential = ServletUtil.newFlow().loadCredential(userId);
+    return new Calendar.Builder(HTTP_TRANSPORT, JSON_FACTORY, credential)
+        .setApplicationName("google.com:clubhub-step-2020")
```

Comment:
nit: Could we make this a constant? I think I made a constant that just had "clubhub-step-2020" as the application name but either way I think this should be a constant in case the full project ID works for both Gmail + Calendar APIs

Reply:
Okay, I can change the application name to clubhub-step-2020 so we can combine the two constants when merging too